src/include/btmem.h

```diff
@@ -830,14 +825,14 @@ struct __wt_ref {
  */
 #define WT_REF_IS_INTERNAL 0x01 /* Page is an internal page */
 #define WT_REF_IS_LEAF 0x02     /* Page is a leaf page */
+#define WT_REF_READING 0x04     /* Page is being read in */
```

Comment:
I think these should be under `AUTOMATIC FLAG VALUE GENERATION START/STOP`

Reply:
Pushed in 990e4eb.